src/db/opt.rs

```diff
@@ -161,4 +166,55 @@ impl AgateOptions {
         // TODO: take other options into account
         self.mem_table_size as u64
     }
+
+    pub fn create(&mut self) -> &mut AgateOptions {
+        self.create_if_not_exists = true;
+        self
+    }
+
+    pub fn num_memtables(&mut self, num_memtables: usize) -> &mut AgateOptions {
```

Comment:
how about using https://github.com/Hoverbear/getset

Reply:
In the early day I don't want to introduce too many dependencies.

Reply:
there are lots of configs, it's verbose to add setter for each of them mannually

Reply:
How about another dedicated PR to do it?

Reply:
okay